Loop/Managers/ServicesManager.swift

```diff
@@ -9,6 +9,7 @@
 import os.log
 import LoopKit
 import LoopKitUI
+import LoopCore
```

Comment:
[nit] intended?

Reply:
Yes, it's for the `@PersistedProperty(key: "Services")` below. PersistedProperty moved to LoopCore